src/main/java/edu/greenblitz/robotName/subsystems/Lifter/LifterConstants.java

```diff
@@ -0,0 +1,21 @@
+package edu.greenblitz.robotName.subsystems.Lifter;
+
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+
+public class LifterConstants {
```

Comment:
add pid for each one (simulation, real) and do the claculate in each subsystem. see pivot in master